scripts/patch/migration-patch-logo.mjs

```diff
@@ -0,0 +1,44 @@
+import {readJSONFile, writeJSONFile} from "../strapi/strapi-api.mjs";
+import {getMigrateIcon, PATCH_SAVE_DEV, PATCH_SAVE_STABLE, writeChainAssetChange, writeChainInfoChange} from "./patch-api.mjs";
+
+const main = async () => {
+  const patchData = await readJSONFile(PATCH_SAVE_DEV);
+  await writeJSONFile(PATCH_SAVE_STABLE, patchData); // init data
+  const {ChainInfo, ChainInfoHashMap, ChainLogoMap, ChainAsset, ChainAssetHashMap, AssetLogoMap} = patchData;
```

Comment:
Can make it simple like that

```
 const migrationData = fs.readFileSync('packages/chain-list-assets/public/patch/0.2.95/preview.json', 'utf-8').replaceAll('https://dev.sw-chain-list-assets.pages.dev', 'https://chain-list-assets.subwallet.app')
```

Then write direct to file with `writeJSONFile` function